### ISSUE TITLE
AddOnSkins - Embed.lua Error

### ISSUE THREAD

**Issue by Noob1e** [ElvUI-WotLK/ElvUI_AddOnSkins#78]
Well, I don't know what's this error but I get the first error when I log into game and the second one when I use Map.

https://postimg.org/image/hj47eebob/

https://postimg.org/image/r3nu1agfv/

**Comment by Loaal** [comment]
6cb217636df8f81398e894011b225805f2301cbc Error should gone.

**Comment by Noob1e** [comment]
Yes, the error is gone, thanks. But now there's another problem. I can't move Skada to the right side. It's always on the left side, even if I choose Omen.

https://postimg.org/gallery/xguwmlp8/

**Comment by Loaal** [comment]
Updated
